docs/test-drive.md

```diff
@@ -1,29 +1,28 @@
 # Take a test drive
 
-You can easily try out Netlify CMS by running it on a pre-configured starter site. Our example in the [intro](/docs/intro) is the [Kaldi small business Hugo template](https://github.com/netlify-templates/kaldi-hugo-cms-template). Use the deploy button below to build and deploy your own copy of the repository:
+Netlify CMS can run in any frontend web environment, but the quickest way to try it out is by running it on a pre-configured starter site with Netlify. Our example here is the [Kaldi coffee company template](https://github.com/netlify-templates/one-click-hugo-cms). Use the button below to build and deploy your own copy of the repository:
 
-[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/netlify-templates/kaldi-hugo-cms-template)
+[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/netlify-templates/one-click-hugo-cms&stack=cms)
 
-## Authenticate with GitHub
+After clicking that button, you’ll authenticate with GitHub and choose a repository name, then Netlify will automatically clone the template and deploy it, bringing you to the site dashboard when the build is complete. Next, you’ll need to set up Netlify's [Identity](https://www.netlify.com/docs/identity) service to authorize users to log in to the CMS.
```

Comment:
Could be it be clearer that Netlify is creating a new repository (with mirrored content) belonging to the authenticated user?